internal/engine/experiment/torsf/testdata/tor.log

```diff
@@ -0,0 +1,31 @@
+Feb 04 15:04:29.000 [notice] Tor 0.4.6.9 opening new log file.
+Feb 04 15:04:29.360 [notice] We compiled with OpenSSL 101010cf: OpenSSL 1.1.1l  FIPS 24 Aug 2021 and we are running with OpenSSL 101010cf: 1.1.1l. These two versions should be binary compatible.
+Feb 04 15:04:29.363 [notice] Tor 0.4.6.9 running on Linux with Libevent 2.1.12-stable, OpenSSL 1.1.1l, Zlib 1.2.11, Liblzma 5.2.5, Libzstd 1.5.2 and Glibc 2.34 as libc.
+Feb 04 15:04:29.363 [notice] Tor can't help you if you use it wrong! Learn how to be safe at https://www.torproject.org/download/download#warning
+Feb 04 15:04:29.363 [warn] Tor was compiled with zstd 1.5.1, but is running with zstd 1.5.2. For safety, we'll avoid using advanced zstd functionality.
+Feb 04 15:04:29.363 [notice] Read configuration file "/home/sbs/.miniooni/tunnel/torsf/tor/torrc-2981077975".
```

Comment:
I think we should add some sanitisation to the log parser so that we don't leak the username in the path in the measurement.

Reply:
Ah nevermind, I see that we are already airing on the side of caution by ignoring anything that doesn't contain the string `Bootstrapped` or the `new bridge descriptor` line, so we should be good.

Reply:
To further clarify: this file is a test fixture containing all my logs to ensure we extract bootstrap logs.